conf/messages.en

```diff
@@ -301,20 +301,26 @@ isImporterVatRegistered.02 = No
 isImporterVatRegistered.checkYourAnswersLabel = Is the importer VAT registered?
 isImporterVatRegistered.error.required = Select yes if the importer is VAT registered
 
-vATPaid.title = How much import VAT was paid to HMRC?
-vATPaid.heading = How much import VAT was paid to HMRC?
-vATPaid.checkYourAnswersLabel = How much import VAT was paid to HMRC?
-vATPaid.hint = This should be the total amount of overpayment for all entries 
-vATPaid.error.required = Enter the amount of import VAT that was paid to HMRC
-vATPaid.error.notANumber = Amount of import VAT that was paid to HMRC must be a number
-vATPaid.error.length = Amount of import VAT that was paid to HMRC must be 99999999999.99 or less
-vATPaid.error.decimalPlaces = Amount of import VAT that should have been paid to HMRC must include pence, like 123.45
-
-vATDueToHMRC.title = How much import VAT should have been paid?
-vATDueToHMRC.heading = How much import VAT should have been paid?
-vATDueToHMRC.checkYourAnswersLabel = How much import VAT should have been paid?
-vATDueToHMRC.error.required = Enter import VAT should have been paid
-vATDueToHMRC.error.length = Import VAT should have been paid must be 14 characters or less
+vatPaid.title = Import VAT overpayment
+vatPaid.checkYourAnswersLabel = Import VAT overpayment
+VATPaid.single_entry.hint = Enter the amount of import VAT that was paid to HMRC, and the amount you think should have been paid. Do not include any agent fees.
+VATPaid.multiple.hint = This should be the total amount of overpayment for all entries. Do not include any agent fees.
```

Comment:
could we be consistent when using keys? VATPaid or vatPaid, single_entry and multiple_entry

Reply:
apologies- keys are now consistent @lmugnai